Author: Enrique Monge
Description: Add CFLAGS, use rights PATHs and INSTALL instead of 'cp' 
--- wmifinfo-0.09.orig/Makefile	2005-03-14 01:28:31.000000000 -0600
+++ wmifinfo-0.09/Makefile	2008-10-28 09:31:44.000000000 -0600
@@ -2,7 +2,7 @@
 # Set this to 'y' if you want support for reading the link quality
 # of nowiresneeded 1148 PCMCIA or Swallow cards
 
-ENABLE_NWN_SUPPORT=n
+ENABLE_NWN_SUPPORT=y
 
 
 ##################################################################
@@ -13,9 +13,11 @@ VERSION=0.09
 
 CC = gcc
 LD = gcc
-COPTS = -Wall -O2 -D'VERSION="$(VERSION)"' -D'NAME="$(NAME)"'
+COPTS = -D'VERSION="$(VERSION)"' -D'NAME="$(NAME)"'
 LDOPTS = -lX11 -L/usr/X11R6/lib -lXpm -lXext
-BINDIR = /usr/local/bin
+BINDIR = $(DESTDIR)/usr/bin/
+INSTALL=/usr/bin/install
+INSTALL_PROGRAM = $(INSTALL) -p    -o root -g root  -m  755
 
 BIN =	wmifinfo
 FILES = wmifinfo.o xutils.o 
@@ -28,7 +30,7 @@ endif
 all:	$(BIN)
 
 .c.o: 	
-	$(CC) $(COPTS) -c $< 
+	$(CC) $(COPTS) $(CFLAGS) -c $< 
 
 $(BIN):	$(FILES)
 	$(LD) -o $@ $(FILES) $(LDOPTS)
@@ -37,7 +39,7 @@ clean:
 	rm -f *.o $(BIN) core ./.#* *.orig *.rej
 
 install:
-	cp $(BIN) $(BINDIR)
+	$(INSTALL_PROGRAM) $(BIN) $(BINDIR)
 
 dist:	clean
 	rm -rf /tmp/wmifinfo-$(VERSION)
